pandas/io/formats/terminal.py

```diff
@@ -46,89 +18,3 @@ def is_terminal():
             return False
         else:  # IPython in a terminal
             return True
```

Comment:
Is this intended to be a public method?  Otherwise, I might even suggest moving this to `pandas.core.config_init.py`, allowing that file to become more self-contained.

Reply:
yeah would do that in a followup ok (or here)

Reply:
I'll save for a follow-up.